src/collections/_documentation/performance/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,192 @@
+- Send a notification to a single legacy integration (service)
+    - "service" is the same as legacy integration
+- Send a notification to a global integration, which includes
+    - [PagerDuty](https://docs.sentry.io/workflow/integrations/global-integrations/#pagerduty)
+    - [Slack](https://docs.sentry.io/workflow/integrations/global-integrations/#slack)
+    - Custom integrations built using the [Integration Platform](https://docs.sentry.io/workflow/integrations/integration-platform/)
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-25_15.28.01.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-25_15.28.01.png)
+
+- Legacy Integrations (via “Service” as any individual integration) that can send Notifications:
+
+    [ ALERT BOX: **Community Integrations** 
+
+    These integrations are [maintained and supported](https://forum.sentry.io/) by the Sentry community. ]
+
+    - Campfire*
+    - IRC
+    - OpsGenie
+    - Pushover
+    - Twilio
+    - VictorOps
+    - [Webhooks](https://docs.sentry.io/webhook-plugin/)
+
+By default, **alert emails** are sent to [LINK: issue owners]. If issue owners isn't configured, or an owner isn't found, the email will do one of two things:
+
+- not send
+- send to all project members depending on the fall-through setting under **Project Settings > Issue Owners**.
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_22.03.07.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_22.03.07.png)
+
+**Rate Limit**
+
+This control allows you to limit the number of times actions are executed for each issue.
+
+- Perform these actions at most once every {interval} for an issue
+    - interval: 5 minutes, 10 minutes, 30 minutes, 60 minutes, 3 hours, 12 hours, 24 hours, one week, or 30 days
+
+For example, if an issue violates alert conditions multiple times in a 30-min period, but your frequency threshold is 30 minutes, you’ll only get one alert.
+
+![https://docs.sentry.io/assets/notifications/alert_frequency-40d4d353342465f52d23147ed2fb32986e1a502c889cfb5579723cd161b53a62.png](https://docs.sentry.io/assets/notifications/alert_frequency-40d4d353342465f52d23147ed2fb32986e1a502c889cfb5579723cd161b53a62.png)
+
+### **Environment**
+
+- All Environments
+- Any one of your defined (and not hidden) environments
+
+![https://docs.sentry.io/assets/notifications/alert_environment-e6e58d46f0a5773deb78f740f1f14ab37f236699161492dc20023d6586504529.png](https://docs.sentry.io/assets/notifications/alert_environment-e6e58d46f0a5773deb78f740f1f14ab37f236699161492dc20023d6586504529.png)
+
+If "All Environments" is selected, the alert conditions are checked individually for each environment, not for the combined events from all environments.
```

Comment:
remove passive "If you select "All Environments", Sentry checks individually for each environment, rather than for combined events across environments", perhaps